### DISCUSSION TITLE
lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/flutter%E6%A1%8C%E9%9D%A2%E7%89%88%E5%BA%93%E9%97%AE%E9%A2%98

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#477] [Announcements]
# lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/flutter%E6%A1%8C%E9%9D%A2%E7%89%88%E5%BA%93%E9%97%AE%E9%A2%98

flutter桌面版库问题 toast 采用bottoast 代理不能使用 http_proxy_override webview没有官方的桌面版和web版 video_player没有官方的桌面版和web版

http://localhost:8080/lx_blog/flutter%E6%8A%80%E6%9C%AF%E7%AF%87/flutter%E6%A1%8C%E9%9D%A2%E7%89%88%E5%BA%93%E9%97%AE%E9%A2%98.html

**Comment by stdroom** [comment]
ddd
